011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/PostgreSqlApplicationResourceTest.java

```diff
@@ -30,10 +29,9 @@
     private ApplicationEntity actualEntity;
     private ApplicationEntity[] actualList;
 
-    @BeforeEach
-    @Transactional
+    @AfterEach
     public void setup() {
```

Comment:
Why not just
```java
    @AfterEach
    public void deleteEntityIfExists() {
        if (actualEntity != null) {
            whenDeleteApplication();
        }
    }
```
? (and `setup` as method name for `@AfterEach` doesn't feel right)

Reply:
Sorry, I tried different things and this was a leftover. Changed it to tearDown.